manual.md

```diff
@@ -49,13 +49,17 @@ RAD Governance Proposals (RGPs) must be formally labeled with the phase of the p
+_Formal Review_ is the last chance for the community to propose changes and give feedback on the proposal before it moves to an on-chain vote. During _Formal Review_, proposal authors are required to publish an updated version of their proposal on the forum taking into account feedback from the *Discussion* phase. If applicable, the proposal must be accompanied by executable on-chain code for the community to review. See this template [INSERT LINK WHEN READY] for how the code should be presented. Proposals in this stage should be posted under the [Active Proposals](https://community.radworks.org/c/governance/proposals/18) category on the forum. 
+
+Each proposal is required to publish a Snapshot poll to gauge consensus on whether it should proceed to an on-chain vote. To initiate a Snapshot poll for a Formal Review, you must be added as an 'author' in our Snapshot space. This measure is in place to prevent spam proposals. Please reach out to Shelby (shelb_ee#9785 on Discord) with the Ethereum address of the party posting your proposal. All Snapshot Polls must begin on **5:00pm GMT+2** the **third Monday of the month** and end at **5:00pm GMT+2** on the **following Monday**. Each Snapshot poll must have three voting options: `Yes` `No` and `Abstain`. Proposal authors should include a link to the Snapshot poll at the top of their proposal on the forum. 
 
 **Submission**
 
-Submitting a proposal on-chain is the final step in the governance process. The proposal should incorporate feedback from the _Discussion_ and _Formal Review_ phases and must be accompanied by executable on-chain code. An RGP can be submitted to on-chain governance by anyone who has an amount of $RAD greater than or equal to **1% of the total $RAD supply** delegated to their address (i.e ≥ 1M $RAD). Please add any executable proposal code to the [proposal code archive](https://github.com/radicle-foundation/radworks-governance/tree/main/proposals). 
+Submitting a proposal on-chain is the final step in the governance process. If proposals pass Formal Review, proposal authors are required to post an updated version of their proposal on the forum and should incorporate feedback from the _Discussion_ and _Formal Review_ phases. The proposal should inlcude links to both the *Discsussion* and the *Formal Review* versions of the proposal. The proposal must be accompanied by executable on-chain code. Please add or update the corresponding proposal code in the [proposal code archive](https://github.com/radicle-foundation/radworks-governance/tree/main/proposals) using this template [INSERT LINK WHEN READY] if not done so already. This version of the proposal should also be posted under the [Active Proposals](https://community.radworks.org/c/governance/proposals/18) category on the forum.
```

Comment:
Same feedback for this. Maybe its not a list of steps, but a "checklist" or something?